test/suites/optional/sriov.robot

```diff
@@ -0,0 +1,74 @@
+*** Settings ***
+Documentation       MicroShift SR-IOV tests
+
+Resource            ../../resources/microshift-process.resource
+
+Suite Setup         Setup Suite With Namespace
+Suite Teardown      Teardown Suite With Namespace
+
+
+*** Variables ***
+${TEMPLATE_PATH}    ${CURDIR}/../../assets/sriov/sriov-network-policy-template.yaml
+${VENDOR_ID}        8086
+${DEVICE_ID}        10c9
+
+
+*** Test Cases ***
+Verify SR-IOV Pods Start Correctly
+    [Documentation]    Waits for pods to enter a running state
+
+    Wait Until Keyword Succeeds    2min    10s
+    ...    All Pods Should Be Running    sriov-network-operator
+
+Create VFs And Verify
+    [Documentation]    Deploys sriovnetworknodepolicy and verifies the VF configuration
+
+    ${pci_address}=    Get PCI Address For Device    ${VENDOR_ID}    ${DEVICE_ID}
```

Comment:
I'd suggest to get the `vendor`, `deviceID` and `pciAddress` from `sriovnetworknodestate`: `oc get sriovnetworknodestate -o yaml -n sriov-network-operator -o json | jq -r '.items[].status.interfaces[]'` You can remove `VENDOR_ID` and `DEVICE_ID` global vars if you use this approach.